frontend/compositions/map/useData.tsx

```diff
@@ -0,0 +1,46 @@
+import { Feature } from "geojson"
+import { useMemo, useState } from "react"
+import uscities from "./uscities.json"
+
```

Comment:
Since this a custom hook, the 'hooks' folder would be the best place for it to live.

Reply:
is this still applicable change to fix?